### ISSUE TITLE
Have applicable output modules join MACB timestamps

### ISSUE THREAD

**Issue by kiddinn** [log2timeline/plaso#306]
When using the mactime parser to parse a bodyfile there are two things that are not happening:
- Two timestamps from the same file and same timestamps are not joined together
- When using the l2tcsv output module the events don't get joined as "M..B"/"MACB", they are all in a row as "M..." and ".A..", etc.
- Filestat parser uses "mtime, atime, etc" for describing timestamps while mactime parser uses "Modification Time", etc. Make them the same

**Comment by kiddinn** [comment]
CL: http://codereview.appspot.com/263830043

**Comment by joachimmetz** [comment]
For reference: http://wiki.sleuthkit.org/index.php?title=Mactime_output

**Comment by Onager** [comment]
Removing as a blocker for the 1.4 release, as we're unlikely to have this done. The addition of the MFT parser has reduced the number of use cases for the mactime parser as well.

**Comment by kiddinn** [comment]
un-assigning myself from this bug

**Comment by kiddinn** [comment]
So just to clarify a bit

Right now the output is something on the lines of:

M...      10-Jan-2012 20:03:19
.A..       10-Jan-2012 20:03:19
..C.      10-Jan-2012 20:03:19
...B      10-Jan-2012 20:03:19

Whereas the request is to make it combined so that it looks more like:

MACB    10-Jan-2012 20:03:19

The same feature applies to the MFT parser, not just the filestat one.

**Comment by joachimmetz** [comment]
@kiddinn can you file a new issue for the UX improvements and clarify what the status of the original issue is since that indicates a parser error.

**Comment by joachimmetz** [comment]
@kiddinn to which output modules this should apply?

**Comment by joachimmetz** [comment]
Per discussion over mail:

```
However there is a pitfall here due to timestamp precision. E.g is atime really the same as the other timestamps? Or was it updated later but due to the precision this information is lost. I think this yields a thorough look.
```

Currently blocking on: https://github.com/log2timeline/plaso/issues/910

**Comment by joachimmetz** [comment]
Upcoming event data changes will finally unblock this.
Grouping implement will be part of the event export / output phase.
WIP: https://github.com/joachimmetz/plaso/tree/outgroup

**Comment by joachimmetz** [comment]
https://codereview.appspot.com/328040043/

**Comment by joachimmetz** [comment]
Changes implemented for l2t_csv format
